src/public/app/widgets/quick_search.js

```diff
@@ -30,35 +30,33 @@ const TPL = `
   </style>
   
   <div class="input-group-prepend">
-    <button class="btn btn-outline-secondary search-button" type="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+    <button class="btn btn-outline-secondary search-button" type="button" data-bs-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
         <span class="bx bx-search"></span>
     </button>
     <div class="dropdown-menu dropdown-menu-left"></div>
   </div>
```

Comment:
Quick search placeholder not visible on dark theme:

<img width="338" alt="image" src="https://github.com/user-attachments/assets/28cf9076-86de-4e05-8a10-719158549475">